freqtrade/optimize/optimize_reports.py

```diff
@@ -509,12 +516,22 @@ def generate_backtest_stats(btdata: Dict[str, DataFrame],
     :param max_date: Backtest end date
     :return: Dictionary containing results per strategy and a strategy summary.
     """
-    result: Dict[str, Any] = {'strategy': {}}
+    result: Dict[str, Any] = OrderedDict([
```

Comment:
wouldn't a simple dict suffice, conisidering they're ordered by default since python 3.6 or 7?

Reply:
Seems like i switched it to dict and back again, opss. Yeah order is also not important, it used to be important with a hacky metadata parsing from same json file as a result (that is no longer there).